python/nav/models/profiles.py

```diff
@@ -1130,7 +1130,7 @@ class MatchField(models.Model):
     # Since we need to know how things are connected this has been done manualy
     FOREIGN_MAP = {
         ARP: 'netbox__arp',
-        CAM: 'netbox__cam',
+        CAM: 'netbox__cam_set',
```

Comment:
Uhm. Here we add set? Why?

Reply:
Because @lunkwill42 originally commented that in https://github.com/Uninett/nav/pull/2423/#discussion_r913782608

Reply:
i.e. https://github.com/Uninett/nav/pull/2423#discussion_r913782608

Reply:
The question is rather: Did this work before, if `cam_set` was the default backrelation name before @johannaengland  made it explicit? :-/